src/Main.java

```diff
@@ -0,0 +1,27 @@
+
+public class Main {
+
+    public static void main(String[] args) {
+        int a = 0;
+        do {a++;
+        if (a % 5 == 0){
+            System.out.println("Buzz");
+        }
+        if (a % 3 == 0) {
+                System.out.println("Fizz");
+            }
+        if (a % 3 == 0 && a % 5 == 0){
+            System.out.println("FizzBuzz");
+            }
+        if (a % 3 != 0 && a % 5 != 0){
```

Comment:
Your code should print only one word instead a number but in your program if 'a' == 15 it prints "Buzz", "Fizz" and "Buzz Fizz". Try to use 'else if' and 'else' blocks